mylasqlanon.py

```diff
@@ -66,24 +66,27 @@
         join_tables = db_config.get(table).get("tables")
         # Go thourhg each table building up the query string
         tmp_cols = []
-        for join_table, join_cols in join_tables.items():
-            for join_col in join_cols:
+        for join_table in join_tables:
+            join_table_name = join_table.get("name")
+            for join_col in join_table.get("cols"):
                 # Get the column name
-                join_name = join_col.get("name")
+                join_col_name = join_col.get("name")
                 # Create a new alias
-                join_alias = join_table + "__" + join_name
-                tmp_cols.append(f"{join_table}.{join_name} AS {join_alias}")
+                join_alias = join_table_name + "__" + join_col_name
+                tmp_cols.append(f"{join_table_name}.{join_col_name} AS {join_alias}")
                 # Update the alias in the column
                 join_col["name"] = join_alias
                 t_config.append(join_col)
         db_cols = ",".join(tmp_cols)
-        db_tables = ",".join(join_tables.keys())
-        db_where = db_config.get(table).get("where")
-        sql = f"SELECT {db_cols} FROM {db_tables} WHERE {db_where}"
+        # Get the name of the first table
+        db_table = join_tables[0].get("name")
+        db_join = db_config.get(table).get("join")
+        sql = f"SELECT {db_cols} FROM {db_table} {db_join}"
     else:
         t_config = (db_config.get(table))
         sql = f"SELECT * from {table}"
-    df = pd.read_sql(sql, engine).infer_objects()
+    logger.info(sql)
```

Comment:
Did you mean to leave the log message in? I think it's fine to at this point, maybe the level could change.